.gitlab-ci-check-docker-build.yml

```diff
@@ -155,3 +158,31 @@ publish:image:mender:dockerhub:
     # while overriding it at the end for Docker Hub, so that the tag/push in script is done for the latter.
     - *export_docker_vars
     - export DOCKER_REPOSITORY=${DOCKER_REPOSITORY#registry.mender.io/}
+
+trigger:saas:sync-staging-component:
+  stage: trigger
+  only:
+    refs:
+      - /^staging$/
+  image: alpine
+  before_script:
+    - *export_docker_vars
+    - *get_release_tool_alpine
+    - apk add --no-cache curl
+  script:
+    # If the repo is not recognized, ignore
+    - if ! echo $(release_tool --list git --all) | grep $CI_PROJECT_NAME; then
+    -  echo "Repository $CI_PROJECT_NAME not found in release_tool. Exiting"
+    -  exit 0
+    - fi
+    # Trigger the sync once per container in this repo
+    - for container in $(release_tool -m git ${CI_PROJECT_NAME} container); do
+    -   echo "Triggering saas staging sync for container $container image $DOCKER_PUBLISH_COMMIT_TAG"
+    -   curl -v -f -X POST
+          -F token=${SAAS_TRIGGER_TOKEN}
+          -F ref=master
+          -F variables[TRIGGER_SYNC_STAGING_COMPONENT]=true
+          -F variables[SYNC_CONTAINER_NAME]=${container}
+          -F variables[SYNC_IMAGE_NAME]=${DOCKER_PUBLISH_COMMIT_TAG}
```

Comment:
Note to self: this is wrong; it should prepend the image name (i.e. `mendersoftware/repo:`)

Reply:
Actually not; we cannot add pass the image name here as it will be different for different containers (`workflows` and `workflows-worker`).

So we need to send only the version part and modify the logic in https://github.com/mendersoftware/saas/pull/327.

I'll fix in the next push.